release/perf_metrics/release_version.txt

```diff
@@ -0,0 +1 @@
+2.22.0
```

Comment:
maybe also use json (as all other files are json)? like a file called `version.json` or `meta.json` ?

which will also make it more expandable, where we can add other meta info into the struct.

Reply:
ohh that sounds better. I updated